plugins/cpp/service/src/filediagram.h

```diff
@@ -3,6 +3,7 @@
 
 #include <service/cppservice.h>
 #include <projectservice/projectservice.h>
+#include <parser/parsercontext.h>
```

Comment:
Unused import.
Even if it was used, it has to go.

Reply:
Fixed in 71ee4563691088569e29c90bd359ac26ca4fc9b6